leftwm/src/config/default.rs

```diff
@@ -199,14 +201,23 @@ impl Default for Config {
             .map(|s| (*s).to_string())
             .collect();
 
+        let scratchpad = ScratchPad {
+            name: "Alacritty".into(),
+            value: "alacritty".to_string(),
+            x: Some(Size::Pixel(860)),
+            y: Some(Size::Pixel(390)),
+            height: Some(Size::Pixel(300)),
+            width: Some(Size::Pixel(200)),
+        };
+
         Self {
             workspaces: Some(vec![]),
             tags: Some(tags),
             layouts: LAYOUTS.to_vec(),
             layout_mode: LayoutMode::Workspace,
             // TODO: add sane default for scratchpad config.
             // Currently default values are set in sane_dimension fn.
```

Comment:
I think what was meant by this TODO was that `fn sane_dimension` is not a good place to set some default values.
Maybe we can clean this up with a fix to #856?

Good catch though.